documents/invoice/model.go

```diff
@@ -487,18 +486,3 @@ func (i *Invoice) CollaboratorCanUpdate(updated documents.Model, collaborator id
 	cf := documents.GetChangedFields(oldTree, newTree, proofs.DefaultSaltsLengthSuffix)
 	return documents.ValidateTransitions(rules, cf)
 }
-
```

Comment:
Are these defined somewhere else? or are not needed.

Reply:
they are not needed as @vedhavyas pointed out to me today